packages/server/graphql/types/UpgradeCTALocationEnum.ts

```diff
@@ -5,15 +5,19 @@ export type UpgradeCTALocationEnumType =
   | 'teamTemplate'
   | 'orgTemplate'
   | 'createNewTemplate'
+  | 'directMeetingLinkLock'
+  | 'timelineHistoryLock'
 
 const UpgradeCTALocationEnum = new GraphQLEnumType({
   name: 'UpgradeCTALocationEnum',
-  description: 'Where the upgrade to pro CTA button was clicked',
+  description: 'Where the upgrade CTA button was clicked',
```

Comment:
I removed `pro` here as we may have different tiers of self service upgrades in the future.